node/pkg/db/governor.go

```diff
@@ -22,29 +22,81 @@ type GovernorDB interface {
 	DeleteTransfer(t *Transfer) error
 	DeletePendingMsg(k *PendingTransfer) error
 	GetChainGovernorData(logger *zap.Logger) (transfers []*Transfer, pending []*PendingTransfer, err error)
+	StoreSignedVAA(v *vaa.VAA) error
```

Comment:
Are these the standard interface that we use, because they don't return errors?

Reply:
The real versions in `node/pkg/db/db.go` can return errors. The versions here are only mocks for testing, so returning errors isn't important.